src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/compilerconfig.json

```diff
@@ -1,130 +1,134 @@
+    "inputFile": "Pages/Components/ColorPicker/BitColorPickerDemo.razor.scss"
+  },
+  {
+    "outputFile": "Pages/Components/List/BitBasicListDemo.razor.css",
+    "inputFile": "Pages/Components/List/BitBasicListDemo.razor.scss"
+  },
+  {
+    "outputFile": "Pages/Components/Modal/BitModalDemo.razor.css",
+    "inputFile": "Pages/Components/Modal/BitModalDemo.razor.scss"
+  },
+  {
+    "outputFile": "Pages/Components/Notification/BitMessageBarDemo.razor.css",
+    "inputFile": "Pages/Components/Notification/BitMessageBarDemo.razor.scss"
+  },
+  {
+    "outputFile": "Pages/Components/Pivot/BitPivotDemo.razor.css",
+    "inputFile": "Pages/Components/Pivot/BitPivotDemo.razor.scss"
+  },
+  {
+    "outputFile": "Pages/Components/Slider/BitSliderDemo.razor.css",
+    "inputFile": "Pages/Components/Slider/BitSliderDemo.razor.scss"
+  },
+  {
+    "outputFile": "Pages/Components/Spinner/BitSpinnerDemo.razor.css",
+    "inputFile": "Pages/Components/Spinner/BitSpinnerDemo.razor.scss"
+  },
+  {
+    "outputFile": "Pages/Components/Toggle/BitToggleDemo.razor.css",
+    "inputFile": "Pages/Components/Toggle/BitToggleDemo.razor.scss"
+  },
+  {
+    "outputFile": "Pages/GetStartedPage.razor.css",
+    "inputFile": "Pages/GetStartedPage.razor.scss"
+  },
+  {
+    "outputFile": "Pages/HomePage.razor.css",
+    "inputFile": "Pages/HomePage.razor.scss"
+  },
+  {
+    "outputFile": "Pages/IconPage.razor.css",
+    "inputFile": "Pages/IconPage.razor.scss"
+  },
+  {
+    "outputFile": "Pages/OverviewPage.razor.css",
+    "inputFile": "Pages/OverviewPage.razor.scss"
+  },
+  {
+    "outputFile": "Shared/MainLayout.razor.css",
+    "inputFile": "Shared/MainLayout.razor.scss"
+  },
+  {
+    "outputFile": "wwwroot/styles/styles.css",
+    "inputFile": "wwwroot/styles/styles.scss"
+  },
+  {
+    "outputFile": "Pages/Components/TextField/BitTextFieldDemo.razor.css",
+    "inputFile": "Pages/Components/TextField/BitTextFieldDemo.razor.scss"
+  },
+  {
+    "outputFile": "Pages/Components/SpinButton/BitSpinButtonDemo.razor.css",
+    "inputFile": "Pages/Components/SpinButton/BitSpinButtonDemo.razor.scss"
+  },
+  {
+    "outputFile": "Pages/Components/DatePicker/BitDatePickerDemo.razor.css",
+    "inputFile": "Pages/Components/DatePicker/BitDatePickerDemo.razor.scss"
+  },
+  {
+    "outputFile": "Pages/Components/Checkbox/BitCheckboxDemo.razor.css",
+    "inputFile": "Pages/Components/Checkbox/BitCheckboxDemo.razor.scss"
+  },
+  {
+    "outputFile": "Pages/Components/RadioButtonList/BitRadioButtonListDemo.razor.css",
+    "inputFile": "Pages/Components/RadioButtonList/BitRadioButtonListDemo.razor.scss"
+  },
+  {
+    "outputFile": "Pages/Components/DropDown/BitDropDownDemo.razor.css",
+    "inputFile": "Pages/Components/DropDown/BitDropDownDemo.razor.scss"
+  },
+  {
+    "outputFile": "Pages/Components/RadioButtonGroup/BitRadioButtonGroupDemo.razor.css",
+    "inputFile": "Pages/Components/RadioButtonGroup/BitRadioButtonGroupDemo.razor.scss"
+  }
```

Comment:
so you are the culprit that changes this file's format incorrectly.
the indentation should be 4 spaces as defined in the `.editorconfig` file.